ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -495,7 +495,7 @@ export default class ConfirmTransactionBase extends Component {
                     <Typography
                       variant={TYPOGRAPHY.H7}
                       fontStyle={FONT_STYLE.ITALIC}
-                      color={COLORS.GRAY}
+                      color={COLORS.UI4}
```

Comment:
This was the only use of `COLORS.GRAY` in the codebase, it fell back to and inherited `COLORS.UI4` already so nothing changed here

Reply:
nice